src/test/java/org/takes/http/FtRemoteTest.java

```diff
@@ -148,7 +142,7 @@ public void exec(final URI home) throws IOException {
                         .body().set("returnsAnEmptyResponseBody").back()
                         .fetch()
                         .as(RestResponse.class)
-                        .assertBody(new IsEqual<>(""))
+                        .assertBody(new IsEqual<>("HTTP/1.1 200 OK\r\n\r\n"))
```

Comment:
@paulodamaso it looks like a head of response, not body. According to test method name the response body should be empty.

Reply:
@g4s8 AssertBody method does not assert just the body of the response. `RsPrint` merges head and body and for that moment on it is treated like a single thing.  So I'm testing if we have just the headers in this response.

Reply:
@g4s8 AssertBody method does not assert just the body of the response. RsPrint merges head and body and for that moment on it is treated like a single thing. So I'm testing if we have just the headers in this response.

Reply:
@paulodamaso I agree with @g4s8. The test now is misleading as it says that `TkEmpty` should return empty body in response, but that's not what you assertion is proven. This assertion proves that we have some bug (which is the fact that response body is not empty even though it should be empty).